roles/openshift_inventory/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- no_log: True  # keep task descriptions legible
+  block:
+
+  - name: Standardize on latest variable names
```

Comment:
I'm glad we're working to automatically standardize variable names but without an explanatory comment block near by I think the reasoning for this fact setting could be lost on the casual reader. Also, clearly expressing what the intended end state is supposed to be would be nice.